AllInOne docs update - Quickstart (#104)
federated-research/docs: website/pages/five_safes_tes/quickstart.mdx

import { Callout, Cards, Tabs, Steps } from "nextra/components";

# Five Safes TES All-In-One Quickstart

<Callout type="warning">
  This is not intended to be a production deployment.
</Callout>

> All-In-One is a demonstration instance of the complete Five Safes TES Weave, intended to be run locally.

Reviewer: ```suggestion
All-In-One is a reference implementation of the complete Five Safes TES Weave, intended to be run locally.
```

@prquinlan could you verify?

The purpose of this Quickstart is allowing you to run all the components of Five Safes TES in a single install, and guide you through sending your first "Hello World" submission to the Five Safes TES.
This then will help you to explore how this weave can be useful for your use case,
or as part of a development pathway.

The quickstart is expected to take approximately 1 hour to complete.

## Prerequisites

- To run the quickstart, you will need [Docker](https://www.docker.com/),
  which can be installed and running on your machine, following these [instructions](https://uomresearchit.github.io/docker-introduction/index.html#installation-of-docker).
  Any issues can be raised [here](https://github.com/federated-research/docs/issues).

> - The Docker compose files of Five Safes TES Weave need to be downloaded or cloned from the [GitHub](https://www.github.com) repository. Doing this by running the following command in your Terminal's working directory:

Reviewer 2: ```suggestion
- The Docker Compose files of Five Safes TES Weave need to be downloaded or cloned from the [GitHub](https://www.github.com) repository. You can do this by running the following command:
```

```
cd AllInOne
```

### Start the Docker Service with `DemoMode = true`

In the same directory as the `docker-compose.yml` file run:

```bash copy
docker compose up -d
```

By default, the docker compose file is configured to run in `DemoMode = true`. This will start the stack with the seeding of some demo data and instances.
For example, `Testing` project will be created, TRE named `DEMO` will be deployed, and users credentials used to communicate between apps will be seeded.

<Callout type="info">
  You can check the status of the services by running `docker stats` in the
  terminal.
</Callout>

<Callout>
Once the `keycloak` service has started, the configuration files for three realms `DARE-Control`, `DARE-TRE` and `Data-Egress` will be imported and applied automatically. 
  As a result, the admin user `globaladminuser` will be created in each realm.

```yaml copy
Username: globaladminuser
Password: password123
```

  </Callout>

### Set up Funnel with Minio

While the All-in-One components are running, Funnel needs to be configured to use the TRE MinIO instance `minioTRE`.

> Follow this [guide](/five_safes_tes/TES/install_funnel) to set up Funnel with Minio.

Reviewer 2: ```suggestion
Follow this [guide](/five_safes_tes/TES/install_funnel) to set up Funnel with MinIO.
```
